frameworks/person-escort-record/README.md

```diff
@@ -0,0 +1,50 @@
+# Person Escort Record framework
+
+This framework contains the sections, steps and questions that need to be completed to produce
+a Person Escort Record (PER) when moving somebody using the Book a secure move service.
+
+The framework contains a set of YAML defintions that will be used by the [Book a secure move API](https://github.com/ministryofjustice/hmpps-book-secure-move-api)
+and [client applications](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend) of the API to generate validation and form journeys.
+
+## Manifest files
+
+Each manifest file contains a section of the PER. Each manifest contains definitions for the steps and the questions that are required to be answered within a tree-like structure.
+
+Manifest files live in `./frameworks/[framework-name]/manifests/`.
```

Comment:
could you add a link as an example

Reply:
I wasn't actually sure about this line because this is within the Readme that already sits within that "framework", so maybe it should just be `./manifests`?

Did you mean a link to the folder too? That's probably a good idea...